README.md

```diff
@@ -29,84 +29,78 @@
 ## Install
 
 ```bash
-npm install --save @webex/components @momentum-ui/react
+npm install --save @webex/components
 ```
 
 ## Usage
-**Webex Components** are built with [Momentum UI](https://github.com/momentum-design/momentum-ui) React components. We need to externally import all the fonts, icons, images and core *CSS* manually. There are two ways to do this:
-
-### 1. CSS
-1. Copy the `@momentum-ui/core/fonts` directory to the 'fonts' directory for your site.
-2. Copy the `@momentum-ui/core/images` directory to the 'images' directory for your site.
-3. Copy the `@momentum-ui/core/css/momentum-ui.min.css` to your styles/css directory.
-4. In the `<head>`, of your HTML reference the location of your momentum-ui.min.css:
-
-    `<link rel='stylesheet' href='path/to/momentum-ui/css/momentum-ui.min.css'>`
-
-***
-### 2. SCSS
-1. Add `$brand-font-folder` and `$icon-font-path` variables to the top of your app's Sass file.
 
+We need to externally import all the fonts, icons, images and core _CSS_ manually. Import `@webex/components/dist/webexComponents.css` into your main entry file. There are two ways to do this:
 
-    ```scss
-    $brand-font-folder: "~@momentum-ui/core/fonts";
-    $icon-font-path: "~@momentum-ui/icons/fonts";
-    ```
+### 1. JavaScript
 
-2. Import `@momentum-ui/core/scss/momentum-ui-components` into your main entry Sass file AFTER the variables.
+```js
+import '@webex/components/dist/webexComponents.css';
+```
 
-    ```scss
-    @import "~@momentum-ui/core/scss/momentum-ui-components";
-    ```
+### 2. Html
 
-3. Compile Sass files using your static compiler or bundler (Webpack, Grunt, Gulp, etc.)
-***
+```html
+<head>
+  <link rel="stylesheet" type="text/css" href="node_modules/@webex/components/dist/webexComponents.css" />
+</head>
+```
 
-Once Momentum styles have been imported into your project, you can start using the components. For example:
+Once Webex styles have been imported into your project, you can start using the components. For example:
 
-  ```js
-  import {WebexAvatar} from '@webex/components';
+```js
+import {WebexAvatar} from '@webex/components';
 
-  function App() {
-    return (
-      <div className='App'>
-        <WebexAvatar title='Webex Avatar' />
-      </div>
-    );
-  }
+function App() {
+  return (
+    <div className="App">
```

Comment:
Why did you change to `"`? There's still `'` used in the imports

Reply:
prettify did that.